ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -121,6 +122,22 @@ export const PageProviderDisplayBehavior = [
         value: false,
       },
 
+      /**
+       * `true` if select all mode is currently enabled. For select all to work, selection must be enabled too.
+       */
+      selectAllEnabled: {
+        type: Boolean,
+        value: false,
+      },
+
+      /**
+       * `true` if select all is active, i.e. all the items are selected.
+       */
+      selectAllActive: {
+        type: Boolean,
+        computed: '_computeSelectAllStatus(_selectAllEnabled, _isSelectAllActive)',
```

Comment:
Not sure where `_selectAllEnabled` is coming from?

Reply:
It should be `selectAllEnabled` and not `_selectAllEnabled`.